groundstation/backend_api/utils.py

```diff
@@ -30,13 +30,13 @@ def decorate(*args, **kwargs):
 
     return decorate
 
-def add_telecommand(command_name, num_arguments, is_dangerous):
+def add_telecommand(command_name, num_arguments, is_dangerous, about_info):
```

Comment:
```suggestion
def add_telecommand(command_name, num_arguments, is_dangerous, about_info=''):
```
Add a blank default about_info so unit tests pass